core/core-impl/src/main/java/ai/timefold/solver/core/impl/score/director/AbstractScoreDirector.java

```diff
@@ -607,7 +621,8 @@ public void assertShadowVariablesAreNotStale(Score_ expectedWorkingScore, Object
                     + "s were triggered without changes to the genuine variables"
                     + " after completedAction (" + completedAction + ").\n"
                     + "But all the shadow variable values are still the same, so this is impossible.\n"
-                    + "Maybe run with " + EnvironmentMode.FULL_ASSERT + " if you aren't already, to fail earlier.");
+                    + "Maybe run with " + EnvironmentMode.FULL_ASSERT_WITH_TRACKING
+                    + " if you aren't already, to fail earlier.");
```

Comment:
Since we're touching this code, let's convert this nastiness to the new formatting pattern.

Remember: if you can, and it is not too disruptive, always leave the code in better shape than the one you found it in.